sdk/storage/Azure.Storage.Blobs/src/BlobClientOptions.cs

```diff
@@ -67,9 +77,20 @@ public enum ServiceVersion
         /// The <see cref="ServiceVersion"/> of the service API used when
         /// making requests.
         /// </param>
+#pragma warning disable AZC0010 // ClientOptions constructors should default ServiceVersion to latest supported service version
         public BlobClientOptions(ServiceVersion version = LatestVersion)
+#pragma warning restore AZC0010 // ClientOptions constructors should default ServiceVersion to latest supported service version
         {
-            Version = version == ServiceVersion.V2019_02_02 ? version : throw Errors.VersionNotSupported(nameof(version));
+            if (version == ServiceVersion.V2019_07_07
+                || version == ServiceVersion.V2019_02_02)
```

Comment:
Nit - we could do a `ServiceVersion.V2019_02_02 <= version && version <= LatestVersion` to avoid updating this code next release.

Reply:
(Can defer)

Reply:
fixed.